source/administration/command-line-tools.rst

```diff
@@ -458,6 +459,40 @@ mattermost command list
     .. code-block:: none
 
        ./mattermost command list myteam
+       
+mattermost command modify
```

Comment:
This is missing from table of contents here https://github.com/mattermost/docs/blob/33d87b8d8e6d8c50deb39631e08ce3fcd556c705/source/administration/command-line-tools.rst#mattermost-command

Reply:
Thanks - fixed.